01-arch/04-object_split.md

```diff
@@ -6,22 +6,27 @@ For each part of the original object's payload, a separate object with own `Obje
 
 ![Large object split](pic/object_split_all)
 
-All objects participating in the split have the `Split` headers set. Depending on the place in the split hierarchy it has different field combinations. There are four possible cases:
+All objects participating in the split have the `Split` headers set. Depending on the place in the split hierarchy it has different field combinations. There are five possible cases:
+
+* Initial object \
+  Initial object that expresses a desire to put a large object in the NeoFS network. It has `split_id` and `parent_header` fields set. The parent header is not finished: since there is no information about payload, it cannot be measured, hashed and signed (also, no object ID can be assigned). However, it has all user information attached to the object (e.g. attributes), and plays the role of representative of the original object until all the parts are uploaded to the NeoFS..
```

Comment:
".." at the end. In some cases size/hash is known.

What about compatibility? We can some big objects already without this part.

Can initial object ID be the big object ID, btw?

Reply:
> ".." at the end. 

fixed

> What about compatibility? We can some big objects already without this part.

should it be described in the spec? you mean to mention that somebody _may_ face such objects?

> Can initial object ID be the big object ID, btw?

not sure, i understand it, can you add some details? do you ask about the situation when the original ID is known?

Reply:
> should it be described in the spec?

How would you deal with an existing large object otherwise? It'd be incompatible with the current specification. Yet it'll be handled somehow by the node.

Reply:
kept both versions